src/alpaca_parser.yrl

```diff
@@ -449,11 +452,20 @@ tuple -> '(' tuple_list ')' :
 
 infix -> term op term : make_infix('$2', '$1', '$3').
 
+literal_fun -> fn terms '->' simple_expr:
```

Comment:
OK. I think that we can restructure the parser in the future to make this a "syntax error" instead of "blacklisting" it.